Source/Concepts.Serialization.Json/ConceptDictionaryConverter.cs

```diff
@@ -87,8 +79,8 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             } 
             catch (Exception ex)
             {
-                _logger.Error($"Error building dictionary: {ex.Message}");
-                throw;
+                if (_logger != null) _logger.Error($"Error reading json: {ex.Message}");
+                throw ex;
```

Comment:
Don't throw ex.  Just throw.  That maintains the correct stacktrace.

Reply:
Ohh, yea